.github/workflows/ci.yml

```diff
@@ -26,6 +26,11 @@ jobs:
           dotnet-version: | 
             6.0.x
             7.0.x
+      - name: Install a SQL Server suite tools
```

Comment:
Do we need any install anything locally to execute this test?

Reply:
Visual Studio usually brings localdb anyway. At least I do not have any sql installations locally and it was running fine.